content/docs/1.0.x/reference/api/index.md

```diff
@@ -59,7 +62,19 @@ The Keptn API is documented in terms of a [Swagger API documentation](https://sw
         link="./assets/swagger-example.png"
         caption="Keptn Swagger API documentation - Example"
         width="700px">}}
-
+        
+## Uploading an API file to a Keptn project 
```

Comment:
The term "API file" doesn't make sense here. It should just be "upload a file to a Keptn project".